uwblueprint/childrens-aid-society
# Add date columns to goals

## Notion ticket link

[Add date columns to goals](https://www.notion.so/uwblueprintexecs/Add-date-columns-to-goals-58deec6643084a089d79bf873d252af1)



## Implementation description
* Added date columns to goals table
* Created the DTO
* Applied migrations



## Steps to test
1. Access database `docker exec -it cas_db /bin/bash -c "psql -U postgres -d cas"`
2. Run `SELECT * FROM goals;` to verify `start_date` and `end_date` columns



## What should reviewers focus on?
* If the migration has been applied correcty.


## Checklist
